server/src/main/java/io/spine/server/commandbus/CommandScheduler.java

```diff
@@ -57,11 +56,9 @@
 
     private boolean isActive = true;
 
-    @Nullable
-    private CommandBus commandBus;
+    private @Nullable CommandBus commandBus;
 
-    @Nullable
-    private Rescheduler rescheduler;
+    private @Nullable Rescheduler rescheduler;
```

Comment:
Why do we introduce this change at all?

What's wrong with the old approach?

Reply:
See [this issue](https://github.com/SpineEventEngine/base/issues/108) on this.